### PR TITLE
Feature new pager

### PR DESCRIPTION
**Pull request by Melissa-Martinez-XL**
## Ticket

* [Changing pager](https://www.notion.so/xmartlabs/Changing-pager-831cd183392f4fdfb1b7a9da6b48e4ab?pvs=4)

## Type of change

* [ ] Fix
* [X] Story
* [ ] Chore

## Description of the change

The pager changed. Now, it only has a number and the next and previous buttons. 

## Screenshot/Execution

https://github.com/xmartlabs/xl-blog/assets/107500715/9493adbc-9682-4baf-8e07-1fe11cb43ba7

## Related PRs
Links to the related PRs

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,6 +45,7 @@
     "gatsby-plugin-slug": "^2.0.0",
     "gatsby-plugin-smoothscroll": "^1.2.0",
     "gatsby-plugin-styled-components": "^6.9.0",
+    "gatsby-plugin-use-query-params": "^1.0.1",
```
[the remark this answers is not in the document]
Comment (Melissa-Martinez-XL) [reply package.json:48]: done

File: src/components/pager/pager.jsx [hunk 2]
```diff
@@ -2,66 +2,44 @@ import React, { useEffect } from "react";
...
+  if (!category && (currentPage - 1) === 1) {
     return '/'
   }
 
-  return `/page/${(currentPage - 1)}`
+  if (category && (currentPage - 1) === 1) {
+    return `/categories/${category}`;
+  }
+
+  return actualCategory() ? `/categories/${actualCategory()}/page/${(currentPage - 1)}` : `/page/${(currentPage - 1)}`;
```
Comment (lauta2912) [inline src/components/pager/pager.jsx:22]: what's the difference between `actualCategory()` and param category?

File: src/helpers/actual-category.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+const actualCategory = (isLayout) => {
+  const filters = [
```
Comment (lauta2912) [inline src/helpers/actual-category.js:4]: use only one filters list. It's not good to replicate constants. You have another list of filters in layout

File: src/components/pager/pager.jsx (hunk 2 above)
Comment (Melissa-Martinez-XL) [reply src/components/pager/pager.jsx:22]: They are the same
Done

File: src/helpers/actual-category.js (hunk 3 above)
Comment (Melissa-Martinez-XL) [reply src/helpers/actual-category.js:4]: Done

File: src/components/pager/pager.jsx [hunk 4]
```diff
@@ -76,7 +53,7 @@ const Pager = ({ pageContext }) => {
       <Link className={currentPage === 1 ? styles.disabledPagerLink : styles.pagerLink} to={prevPage} rel="prev">
         ← Prev
       </Link>
-      <Pages data={pagesData} />
+        <Link to={pagesData.currentPage} className={styles.linkNumber}>{pagesData.currentPage}</Link>
```
[the remark this answers is not in the document]
Comment (Melissa-Martinez-XL) [reply src/components/pager/pager.jsx]: Done

**Review by lauta2912 — APPROVED** [review]

**Pull request merged**